unlock-js/src/FastJsonRpcSigner.js

```diff
@@ -0,0 +1,75 @@
+import { ethers } from 'ethers'
+import utils from './utils'
+
+/**
+ * This file is only needed with ethers v4. v5 will come with an UncheckedJsonSigner
+ * that we can use.
+ *
+ * See https://github.com/ethers-io/ethers.js/issues/511
+ */
+export default class FastJsonRpcSigner extends ethers.Signer {
```

Comment:
This file is tested by the existing tests. If it changed behavior at all, they would fail (and did for many attempts in testing until I got the thing right)